### DISCUSSION TITLE
repository_dispatch action gets cancelled due to higher priority merge events

### DISCUSSION THREAD

**Discussion by sitkumar** [community/community#44753] [Actions]
### Select Topic Area

Bug

### Body

Dispatch Action merge events have a limit of concurrency queue for 4-5 queued pipelines.
Once it exceeds that value(more than 4 repository_dispatch actions), the queued pipeline with lower priority gets cancelled

I am using peter-evans/repository-dispatch@v2 and given concurrency parameters as well but seems it is not working for me.

      - name: Repository Dispatch
        uses: peter-evans/repository-dispatch@v2
        with:
          token: ${{ secrets.PAT }}
          repository: username/my-repo
          event-type: merge-event
          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
       
concurrency:
  group: shared-services
  cancel-in-progress: false

**Comment by jge162** [comment]
@sitkumar hm, try GITHUB_TOKEN to create a new repository dispatch event using the REST API, this way you can set the Accept header to application/vnd.github+json and check the status of the previous repository dispatch event and decide if you want to proceed or not.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sitkumar ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
